### PR TITLE
delete trainee for future challenges

### PR DESCRIPTION
**Pull request by vineetg3**

### CODE REVIEW

File: app/assets/javascripts/application.js
Comment (satyabhavsar1) [inline app/assets/javascripts/application.js:1]: Do we need this file?

File: app/views/challenges/trainees_list.html.erb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <%= render 'main/navbar' %>
+<%= javascript_include_tag 'application' %>
```
Comment (satyabhavsar1) [inline app/views/challenges/trainees_list.html.erb:6]: Are we using it for something?

File: app/assets/javascripts/application.js
Comment (vineetg3) [reply app/assets/javascripts/application.js:1]: Yea its used to enable JS dependent actions. "//" are not comments.

File: app/views/challenges/trainees_list.html.erb (hunk 1 above)
Comment (vineetg3) [reply app/views/challenges/trainees_list.html.erb:2]: removed it

Comment (vineetg3) [reply app/views/challenges/trainees_list.html.erb:6]: Yup for my feature, link_to button

**Review by satyabhavsar1 — APPROVED** [review]

**Pull request merged**